### PR TITLE
Add detailed Linux client installation instructions to README

### PR DESCRIPTION
**Pull request by jtbg**
## Updates to Linux client installation instructions:
- moves the Linux server instructions above the Linux client instructions.  The former is more commonly used (every Linux client has a Linux server, but not every Linux server has Linux clients)
- adds recommendation to install before other mods
- adds screenshots for locating game folder and adding launch instructions in Steam
- refactors to use bash commands for archive extraction and permissions, as this is more consistent across distributions 
- adds a backup of user's existing game data folder

### CODE REVIEW

**Comment by jtbg** [comment]
If this PR is useful, I can add update the install instructions for Windows and servers and/or package up the commands into an interactive shell script

File: README.md [hunk 1]
```diff
@@ -333,6 +322,34 @@ This guide does not cover how to create a dedicated server. These are the steps
...
 6. Make sure to start the server via start_server_bepinex.sh, else the mod will not be loaded.
 
+### **Game[Linux/Unix]**
+
+1. **NOTE:** It is recommended to install ValheimPlus before installing other mods, especially those which also use BepInEx.
+2. Download the [latest package called UnixServer.zip over this link](https://github.com/valheimPlus/ValheimPlus/releases/latest/). *(Scroll down and click "assets")*.
+3. Locate your `Valheim/` game folder.  
+  Open your Steam client right-click the Valheim game in your Steam Library.  
+  Select *"Manage 🡺 Browse local files"*, and make note of the location.  
+  ![image](https://user-images.githubusercontent.com/10716475/142159998-38105800-6f3a-4229-9140-02c363d16ee4.png)
```
Comment (nxPublic) [inline README.md:332]: This Image must be added to this PR and then properly linked.

File: README.md [hunk 2]
```diff
@@ -333,6 +322,34 @@ This guide does not cover how to create a dedicated server. These are the steps
...
+  ```
+  mkdir $my_path/../Valheim.`date +%Y-%m-%d` && cp -br $my_path/* $_ && echo "backup created"    
+  unzip -o ~/Downloads/UnixServer.zip -d $my_path  
+  chmod u+x $my_path/start_game_bepinex.sh
+  ```
+  
+10. Return to your Steam client. Right-click the Valheim game in your Steam library. Select *"Properties...."*  
+  In the pop-up, add the following to the *"Launch Options"* `./start_game_bepinex.sh %command%`  
+  (in rare cases that this command does not work, use `bash /start_game_bepinex.sh %command%` instead)  
+   ![image](https://user-images.githubusercontent.com/10716475/142167891-f54c273d-d438-4bd0-ab02-b66c79b199cc.png)
```
Comment (nxPublic) [inline README.md:350]: This Image must be added to this PR and then properly linked.

**Review by nxPublic — CHANGES_REQUESTED** [review]

**Comment by nxPublic** [comment]
Please add the missing images to the PR and its merge ready!
Great Work!

The lack of Unix/Linux information is not for a lack of willingness. Its because of a lack of knowledge, I never use Linux further than a command line prompt. If you would like to help improving the mod description or instructions further, feel free to slip into my DM's in [Discord ](https://discord.valheim.plus) any time, they're open.

**Review by nxPublic — APPROVED** [review]

**Pull request merged**